docs/source/core/core.rst

```diff
@@ -287,9 +287,7 @@ A NeMo configuration file should look similar to the following:
         decoder:
             ...
 
-More specific details about configuration files for each collection can be found on the following pages:
-
-:ref:`NeMo ASR Configuration Files`
+For more information about configuration files for each collection, refer to :ref:`NeMo ASR Configuration Files` section.
```

Comment:
We should add additional links to configuration file info pages of the other tasks, as well. We have ASR, speaker diarization, and speaker recogition so far.
(Also, this is the aforementioned wrong link due to an incorrect header on the other page. This link should go to the ASR config files docs but instead leads to the speaker recognition docs, which can be fixed by updating the speaker recognition page title.)